app/models/alchemy/element.rb

```diff
@@ -305,6 +305,21 @@ def copy_nested_elements_to(target_element)
       end
     end
 
+    # List element contents by the definition in elements.yml
+    def defined_contents
+      existing_contents = contents
+      editable_contents = []
+      definition[:contents].each do |defined_content|
```

Comment:
We do not want the elements.yml to be the conical source of contents. In the future elements and contents get created via multiple sources (first and foremost the API).

Reply:
But some solution is needed for new element fields to be added and displayed after saving an element. They have to be recreated or edited in the database now.

Reply:
In older versions in this case a notice appeared, telling 'no editor field is present', because I always forgot to make one.

Reply:
Right. We could display this notice again whenever a content is defined, but not created yet. Keeping the possibility to add new contents to the blueprint, but still only load the contents from the database. WDYT? Want to give it a try?

Reply:
Yes, soon.